resizablecsslayout/assembly/assembly.xml

```diff
@@ -0,0 +1,18 @@
+<assembly xmlns="http://maven.apache.org/ASSEMBLY/2.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/ASSEMBLY/2.0.0 http://maven.apache.org/xsd/assembly-2.0.0.xsd">
+  <!-- TODO: a jarjar format would be better -->
+  <id>jar-with-dependencies</id>
```

Comment:
Not sure how adding `assembly.xml` relates to switching to Java 8, but I do notice that it is missing currently. Also I have no idea nor time to verify what `jar-with-dependencies` does, I'll presume it does

Reply:
Adding `assembly.xml` doesn't relate to this issue, but cannot build without it :) I'm no expert on assembly plugin, I just chose `jar-with-dependencies` as the only option that has `<format>jar</format>` on this list: https://maven.apache.org/plugins/maven-assembly-plugin/descriptor-refs.html